Carmunication/Controllers/LoginViewController.swift

```diff
@@ -10,29 +10,81 @@ import CryptoKit
 import FirebaseAuth
 import UIKit
 
-// 애플 로그인 파이어베이스 인증 시 재전송 공격을 방지하기 위해 요청에 포함시키는 임의의 문자열 값
-fileprivate var currentNonce: String?
-
 class LoginViewController: UIViewController {
```

Comment:
상속하지 않는 클래스에 대해서는 final 키워드를 사용하는 것을 권장합니다 !
```suggestion
final class LoginViewController: UIViewController {
```